def addNumber(n1,n2):
  result = n1+n2
  print(result)
  
addNumber(2,4)
  
  
def add(a,b):
  ans = a + b


# add(2,4)
# print(ans); # throw an error why? ans is local
  
    